packages/design-picker/src/utils/available-designs.ts

```diff
@@ -2,9 +2,9 @@ import { isEnabled } from '@automattic/calypso-config';
 import { shuffle } from '@automattic/js-utils';
 import { addQueryArgs } from '@wordpress/url';
 import { availableDesignsConfig } from './available-designs-config';
-import type { MShotsOptions } from '../components/mshots-image';
 import type { Design, DesignUrlOptions } from '../types';
 import type { AvailableDesigns } from './available-designs-config';
+import type { MShotsOptions } from '@automattic/onboarding';
```

Comment:
The order of the type import seems to be weird and is nor consistent with the order of the function import 🤔

Reply:
Thanks for spotting. Maybe it slipped in during a rebase when prettier wasn't able to run.

Reply:
Nope, the auto-fixer is leaving this import where it is for some reason 🤷 
Maybe it's because `import type` statements always appear below `import` statements. But then I would have at least expected it to be the very first `import type`. I don't want to fight the import order auto-fixer though.

Reply:
Yeah, we can just leave it there.